src/ResourceManager/Automation/Commands.Automation/Cmdlet/StartAzureAutomationDscCompilationJob.cs

```diff
@@ -64,7 +79,7 @@ protected override void AutomationProcessRecord()
                                               Resources.ConfigurationDataShouldNotBeInJobParameters, "-ConfigurationData"));
             }
 
-            job = this.AutomationClient.StartCompilationJob(this.ResourceGroupName, this.AutomationAccountName, this.ConfigurationName, this.Parameters, this.ConfigurationData);
+            job = this.AutomationClient.StartCompilationJob(this.ResourceGroupName, this.AutomationAccountName, this.ConfigurationName, this.Parameters, this.ConfigurationData, this.IncrementNodeConfigurationBuild);
```

Comment:
I have tested this with a locally build installer. It works because the current getter of the switch parameter returns the boolean value (this._incrementNodeConfigurationBuild) as specified above.

I'll change the code (as you wrote above - removing the need of a separate value) and with that the IsPresent flag should help.